packages/backend-defaults/src/entrypoints/auditor/auditorServiceFactory.ts

```diff
@@ -32,17 +32,16 @@ import { DefaultRootAuditorService } from './Auditor';
 export const auditorServiceFactory = createServiceFactory({
   service: coreServices.auditor,
   deps: {
-    rootLogger: coreServices.rootLogger,
+    logger: coreServices.logger,
     auth: coreServices.auth,
     httpAuth: coreServices.httpAuth,
     plugin: coreServices.pluginMetadata,
   },
-  async createRootContext({ rootLogger }) {
-    const auditor = DefaultRootAuditorService.create({ rootLogger });
-
-    return auditor;
-  },
-  factory({ plugin, auth, httpAuth }, rootAuditor) {
-    return rootAuditor.forPlugin({ auth, httpAuth, plugin });
+  factory({ logger, plugin, auth, httpAuth }) {
+    const auditLogger = logger.child({ isAuditorEvent: true });
```

Comment:
```suggestion
    const auditLogger = logger.child({ isAuditEvent: true });
```

does that name make more sense?

Reply:
Yep I think it does, #28575